### DISCUSSION TITLE
Can you support a lower version of glibc？

### DISCUSSION THREAD

**Discussion by alongnice** [spacedriveapp/spacedrive#1682] [Community Support]
I use uos Professional Edition operating system, which is a Chinese linux system, currently does not support GLIBC_2.29, but I am very interested in this project and want to try.

**Comment by brxken128** [comment]
We're already using Ubuntu 20.04 within our release workflow IIRC - i don't think Github even offer any lower, and builds would likely have to be prepared manually on an older system (or possibly even just within a container).

I have successfully built for `glibc 2.29` before, but it was a one-off quite a few months ago - I used the official Rust containers and it worked great, but within CI it may not be _as_ simple sadly.

I doubt we make use of anything specific to more recent versions of `glibc` anyway, but without solid Github runner support we'd have to build within a container, which would make Linux builds take **substantially** longer (and they already take quite a while).

**Comment by HeavenVolkoff** [comment]
@alongnice What distro and version exactly are you using? I couldn't find anything online about `Professional Edition operating system`. Could you share a link to its homepage or a [DistroWatch](https://distrowatch.com/search.php) page about it?
